pkg/resolvers/endpoints.go

```diff
@@ -491,3 +497,22 @@ func dedupPorts(policyPorts []policyinfo.Port) []policyinfo.Port {
 	}
 	return nil
 }
+
+func (r *defaultEndpointsResolver) getPodList(ctx context.Context, opts client.ListOption) ([]corev1.Pod, error) {
```

Comment:
Maybe to make this more clear, it can be `getNonHostNetworkingPodList` ?

Reply:
updated the name.